eigrpd/eigrp_northbound.c

```diff
@@ -242,8 +242,9 @@ static int eigrpd_instance_active_time_modify(struct nb_cb_modify_args *args)
 {
 	switch (args->event) {
 	case NB_EV_VALIDATE:
+		zlog_warn("active time not implemented yet");
```

Comment:
Please log to `args->errmsg` instead so that the error messages will be returned to the CLI users.

Example: `snprintf(args->errmsg, args->errmsg_len, ...);`

Please see #6414 for more details.

Reply:
fixed